app/views/posts/index.html.erb

```diff
@@ -1,3 +1,46 @@
-<h1>Posts#index</h1>
-<h1>Here is a list of posts for a given user</h1>
-<p>Find me in app/views/posts/index.html.erb</p>
\ No newline at end of file
+<div class='all-posts-body'>
+    <div class='u-detail-top'>
+        <div class='u-d-img'>
+            <%= image_tag 'image-placeholder.png', width: '150px', height: '150px' %>
+        </div>
+        
+        <div class='u-detail-user'>
+            <h1 class='u-detail-user-name'>
+                <%= @user.name  %>
+            </h1>
+            <p class='u-detail-user-counter'>
+                Number of posts: <%= @user.posts_counter %>
+            </p>
+        </div>
+    </div>
+
+    <div class='u-detail-rec-posts'>
+        <% @posts.each  do |post| %>
+            <div class='u-detail-rec-posts-card'>
+                <h2>
+                    <%= link_to "#{post.title}", user_post_path(@user.id, post.id), class: 'post-link' %>
+                </h2>
+                <p>
+                    <%= post.text %>
+                </p>
+
+                <div class='com-lik-counters'>
+                    <p>Comments: <%= post.comments_counter %></p>
+                    <p>Likes: <%= post.likes_counter %></p>
+                </div>
+            </div>
```

Comment:
- [ ] Kindly make sure you display the real number of your comments for a post. Check here :point_down: 

![image](https://user-images.githubusercontent.com/65068771/155689630-335977c9-d2a2-4458-a1dd-d609a4a5dfac.png)

Reply:
I have taken a screenshot and the correct number of comments are being displayed.
![Rails capture](https://user-images.githubusercontent.com/63917725/155705711-64e089e9-bddc-4603-b6e4-86c8c660f65e.PNG)